CharlieBackend.Api/Controllers/SchedulesController.cs

```diff
@@ -40,15 +40,6 @@ public async Task<ActionResult<List<ScheduleDto>>> GetAllSchedules()
             return Ok(await _scheduleService.GetAllSchedulesAsync());
         }
 
```

Comment:
SHould not use OK, but should operate with Result.GetSuccess object and converted in to ToActionResult()